src/parsing.jl

```diff
@@ -317,9 +322,15 @@ for pq_eltype in ("int2", "int4", "int8", "float4", "float8", "oid", "numeric")
     jl_eltype = _DEFAULT_TYPE_MAP[Symbol(pq_eltype)]
 
     # could be an OffsetArray or Array of any dimensionality
-    _DEFAULT_TYPE_MAP[array_oid] = AbstractArray{jl_eltype}
+    _DEFAULT_TYPE_MAP[array_oid] = AbstractArray{Union{jl_eltype, Missing}}
+
+    @eval function Base.parse(::Type{AbstractArray{Union{$jl_eltype, Missing}}},
+                              pqv::PQValue{$array_oid})
```

Comment:
This is my preferred indentation style:

```julia
    @eval function Base.parse(
        ::Type{AbstractArray{Union{$jl_eltype, Missing}}}, 
        pqv::PQValue{$array_oid},
    )
```

Reply:
Mine too but I recently got chided by the folks at GLM.j for not using the "Julia" indentation style... which is supposedly keeping args on the same line line as parens?

Reply:
Well you'll be at home in Invenia Land :)